test/e2e/adminapi_resources.go

```diff
@@ -58,6 +59,10 @@ var _ = Describe("[Admin API] List Azure resources action", func() {
 		vnet, err := clients.VirtualNetworks.Get(ctx, r.ResourceGroup, r.ResourceName, "")
 		Expect(err).NotTo(HaveOccurred())
 
+		diskEncryptionSet, err := clients.DiskEncryptionSets.Get(ctx, vnetResourceGroup, fmt.Sprintf("%s-disk-encryption-set", vnetResourceGroup))
+		Expect(err).NotTo(HaveOccurred())
+		expectedResourceIDs = append(expectedResourceIDs, strings.ToLower(*diskEncryptionSet.ID))
+
```

Comment:
This solves work item [12050816](https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/12050816). I assigned it to you.